### PR TITLE
37 css responsive

### PR DESCRIPTION
**Pull request by joelmayemba**
Navbar update with an active on a href + responsive page for contact

### CODE REVIEW

File: public/assets/js/navbar.js
Comment (Benja-san) [inline public/assets/js/navbar.js:1]: replace var by const or let

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: public/assets/js/navbar.js
Comment (joelmayemba) [reply public/assets/js/navbar.js:1]: Done

**Review by chouaibj — APPROVED** [review]

File: src/View/Mocktails/index.html.twig
Comment (Benja-san) [inline src/View/Mocktails/index.html.twig:1]: add H1 to pages when they need referencing

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — CHANGES_REQUESTED** [review]
Why did you change let into var ?

**Review by Benja-san — APPROVED** [review]

**Pull request merged**